oat-sa/extension-tao-system-status
# feat: added support to check aurora AcuUtilization instead rds free s…

Ticket - https://oat-sa.atlassian.net/browse/REL-917

## Goal
Support has been added to monitoring the Aurora AcuUtilization system when the program works with it.

## Changelog
- feat: added support to check aurora AcuUtilization instead rds free space

## How to test
1. Open Tools -> System status page to see the system status monitor
2. There is 'Used space on RDS storage' for RDS instances and 'ACU Utilization on RDS storage' for Aurora cluster instances
3. You can check unit tests for Aurora system status checker `AwsRDSAcuUtilizationCheckTest`
